deps.bzl

```diff
@@ -358,14 +359,14 @@ genrule(
   cmd = '''
     DIR=$$(dirname $(execpath _config.yml))
     $(execpath @jekyll_nix//:bin/jekyll) build -s $$DIR
-    tar hc _site \\
-        --owner=1000 \\
-        --group=1000 \\
-        --mtime=2000-01-01\\ 00:00Z \\
-        --no-acls \\
-        --no-xattrs \\
-        --no-selinux \\
-        --sort=name \\
+    tar hc _site \\\\
```

Comment:
This was an invalid escape sequence before which Bazel implicitly translated to this. Now it errors out on this.